ex03.ex

```diff
@@ -101,7 +114,15 @@ defmodule Ex03 do
 
   """
 
-  def list_equal . . . "your code"
+  defmodule Sample do
```

Comment:
I was writing all code in a separate Sample file which started with defmodule Sample do and when it worked, I mistakenly copied the module too instead of simply copying the content of the module.